Source/DafnyLanguageServer/Handlers/DafnyCodeActionHandler.cs

```diff
@@ -193,8 +193,13 @@ public class DafnyCodeActionInput : IDafnyCodeActionInput {
   }
 
   public string Extract(Range range) {
+    if (!CodeLineToPos.ContainsKey(range.Start.Line)) {
```

Comment:
Not related to this PR, but why did we add `CodeLineToPos` to `CodeActionHandler` instead of to `DocumentTextBuffer` ? It seems like quite a piece of functionality that many pieces of the IDE are interested in.